agent/status.go

```diff
@@ -109,15 +109,15 @@ func (a *Agent) statusHandler() http.HandlerFunc {
 }
 
 type triggerEndTaskResp struct {
-	Description            string   `json:"desc,omitempty"`
-	Status                 string   `json:"status,omitempty"`
-	Type                   string   `json:"type,omitempty"`
-	AddFailureMetadataTags []string `json:"add_failure_metadata_tags,omitempty"`
+	Description            string   `json:"desc"`
+	Status                 string   `json:"status"`
+	Type                   string   `json:"type"`
+	AddFailureMetadataTags []string `json:"add_failure_metadata_tags"`
```

Comment:
The ticket is specifically non-pointer struct tags rather than all types, I made [this playground example](https://go.dev/play/p/tnmLl6lLKZv) which shows it still does effect these types

Reply:
Oops my github just updated to the new commits, my bad!